scylla/src/transport/load_balancing/dumb.rs

```diff
@@ -0,0 +1,14 @@
+use super::{LoadBalancingPolicy, Plan, Statement};
+use crate::transport::cluster::ClusterData;
+
+pub struct DumbPolicy;
```

Comment:
We need an explanation for its existence (and a commit message), and strictly speaking this policy may not be "dumb" for certain workloads. We can call it "SimplePolicy" or "AllNodesPolicy".